packages/opentelemetry/src/spanExporter.ts

```diff
@@ -197,20 +154,15 @@ export class SentrySpanExporter {
         bucketEntry.spans.delete(span);
       }
     }
+    // Cancel a pending debounced flush, if there is one
+    this._debouncedFlush.cancel();
   }
 
   /** Clear the exporter. */
   public clear(): void {
     this._finishedSpanBuckets = this._finishedSpanBuckets.fill(undefined);
-    this._clearTimeout();
-  }
-
-  /** Clear the flush timeout. */
-  private _clearTimeout(): void {
-    if (this._flushTimeout) {
-      clearTimeout(this._flushTimeout);
-      this._flushTimeout = undefined;
-    }
+    this._sentSpans.clear();
```

Comment:
Don't know the expected semantics around `clear()` but calling this method feels quite dangerous as it causes data loss.

Reply:
yes, that is expected. this is more used in tests, and when the span processor is shut down, so this should be fine!